docs-ref-services/cosmos-readme.md

```diff
@@ -1,18 +1,4 @@
----
-title: Azure Cosmos DB SQL API client library for Python
-keywords: Azure, python, SDK, API, azure-cosmos, azure-cosmos
-author: maggiepint
-ms.author: magpint
-ms.date: 05/28/2020
-ms.topic: article
-ms.prod: azure
-ms.technology: azure
-ms.devlang: python
-ms.service: cosmos-db
----
-
-# Azure Cosmos DB SQL API client library for Python - Version 4.0.0 
-
+# Azure Cosmos DB SQL API client library for Python
```

Comment:
Make certain to leave the version here. ` - Version 4.0.1` or so.

Reply:
Reverted